app/main/forms/auth_forms.py

```diff
@@ -62,6 +62,15 @@ class CreateUserForm(Form):
         ]
     )
 
+    phone_number = StringField(
+        'Phone number', id="input_phone_number",
+        validators=[
+            Regexp("^$|^\\+?([\\d\\s()-]){6,20}$",
+                   message="Phone numbers must be between 6 and 20 characters."
```

Comment:
Hey,

Let's maybe update to 21 (characters includes digits and + sign doesn't it?). It's phrased like this to be consistent with the other error messages on the page. I'd like to update all the messages on this page so they're active, eg 'You must', but that felt like a separate story and didn't want there to be mission creep. Can talk through alternatives on Monday.